irr/include/CMeshBuffer.h

```diff
@@ -137,7 +137,7 @@ class CMeshBuffer final : public IMeshBuffer
 	//! Index buffer
 	SIndexBuffer *Indices;
 	//! Bounding box of this meshbuffer.
-	core::aabbox3d<f32> BoundingBox;
+	core::aabbox3d<f32> BoundingBox{-1.0f, -1.0f, -1.0f, 1.0f, 1.0f, 1.0f};
```

Comment:
empty mesh buffers do not have anything inside, you can safely replace this:
```suggestion
	core::aabbox3d<f32> BoundingBox{{0, 0, 0}};
```

Reply:
It's still possible to get the bounding box of an empty mesh buffer. That'd be a behavior change.

Reply:
I get that you wrote this PR in a way that it changes exactly zero behavior, but there is no need to be unnecessarily overly cautious.

Reply:
I have another PR open already which is an appropriate place to put this. There's no need to unnecessarily put behavior changes here when the other PR exists anyway.

Reply:
It's a perfectly valid thing to do but returning a zero-ed box for an empty buffer is still more correct than returning some "random" 2x2x2 cube.